docs/releases/202311.mdx

```diff
@@ -0,0 +1,10 @@
+---
+id: 202311
+title: November 2023
+displayed_sidebar: releaseSideBar
+---
+
+### November 2 2023
+**Features/Updates**
+- Flows can now be assigned to specific users so that a curated list of My Flows appears on the home page. Admins can assign Flows to users from the Flow settings panel, or users can self-assign Flows from the All Flows tab.
```

Comment:
The new look of the home page is very different; this might want to be a bit stronger to reflect that. (That is, it's a change a user sees regardless of whether anyone has assigned them a flow).